### PR TITLE
bump to include library-go and apps consts

### PR DESCRIPTION
**Pull request by mfojtik**
Need to replace `pkg/git` and also include the consts for apps. Will follow up with PR's to accommodate the changes in origin code. 

/cc @bparees 
/cc @deads2k

### CODE REVIEW

**Comment by deads2k** [comment]
I expected a straightforward swap of values here.

**Comment by mfojtik** [comment]
@deads2k done.

**Comment by deads2k** [comment]
please squash "change import for pkg/git to library-go" into "remove pkg/git"

/lgtm

**Comment by bparees** [comment]
s2i changes look fine. and thanks for taking care of the git package.

**Comment by mfojtik** [comment]
squashed, reapplying lgtm label

File: pkg/apps/controller/deploymentconfig/deploymentconfig_controller.go [hunk 1]
```diff
@@ -436,49 +436,49 @@ func calculateStatus(config *appsv1api.DeploymentConfig, rcs []*v1.ReplicationCo
...
-		case appsutil.DeploymentStatusRunning:
+		case appsv1.DeploymentStatusRunning:
 			if appsutil.IsProgressing(config, newStatus) {
-				appsutil.RemoveDeploymentCondition(newStatus, appsv1api.DeploymentProgressing)
+				appsutil.RemoveDeploymentCondition(newStatus, appsv1.DeploymentProgressing)
 				msg := fmt.Sprintf("replication controller %q is progressing", latestRC.Name)
-				condition := appsutil.NewDeploymentCondition(appsv1api.DeploymentProgressing, v1.ConditionTrue,
-					appsutil.ReplicationControllerUpdatedReason, msg)
+				condition := appsutil.NewDeploymentCondition(appsv1.DeploymentProgressing, v1.ConditionTrue,
+					string(appsv1.ReplicationControllerUpdatedReason), msg)
```
Comment (tnozicka) [inline pkg/apps/controller/deploymentconfig/deploymentconfig_controller.go]: @mfojtik why is this changing types?

**Pull request merged**